backend/db/migrate/20231128121022_update_priority_landscape_translations.rb

```diff
@@ -0,0 +1,11 @@
+class UpdatePriorityLandscapeTranslations < ActiveRecord::Migration[7.0]
+  def up
+    Rake::Task["priority_landscapes:translate"].execute
+    Location.priority_landscape.update_all visible: true
```

Comment:
question: are these new priority landscapes already in the locations table?

Reply:
Yes, all priority landscapes are already there -- can be verified at https://staging.hecoinvest.org/backend/backoffice/projects when you check options at priority landscape filter.